oci-pipeline-examples/oci-java-micronaut-devops-example/gradle.properties

```diff
@@ -0,0 +1 @@
+micronautVersion=1.3.4
```

Comment:
Might include a note here, suggesting that people check for and use the latest-and-greatest version (as this will go stale over time).